Maple2.Server.Game/PacketHandlers/RequestCubeHandler.cs

```diff
@@ -778,5 +841,76 @@ private static bool IsCoordOutsideArea(Vector3B position, Home home) {
 
         return false;
     }
+
+    public static void RequestLayout(GameSession session, HomeLayout layout, TableMetadataStorage tableMetadata) {
```

Comment:
This is public static so the code can be reuse in [Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs](https://github.com/AngeloTadeucci/Maple2/pull/239/files#diff-b8131e957e10899fa6f81f1092305d6d926b46f5097f47a7f5c9c5e12cee5bbb).

It can be moved to HousingManager but would need all other helpers to be moved as well.

Reply:
I think that'd be best. Housing manager sounds like it'll be needed.